tests/test_slide.py

```diff
@@ -501,6 +501,20 @@ def it_can_add_a_new_slide(self, add_fixture):
         assert slides._sldIdLst.xml == expected_xml
         assert slide is slide_
 
+    def it_can_duplicate_a_slide(self, slide_fixture):
+        from pptx import Presentation
+        prs = Presentation()
+        prs.slides.add_slide(prs.slide_layouts[1])
+        slides = slide_fixture[0].slides
```

Comment:
I'm finding the implementation a bit complicated, and what I'm attempting based on what I see here is throwing an error.

```p = Presentation()
p = Presentation('../data/feb_temp.pptx')
p2 = Presentation('../data/new_charts_tables.pptx')
dupe = p.slides.duplicate_slide(p2.slides[0])
```
I'm consistently getting: `AttributeError: 'property' object has no attribute 'iter_shape_elms'`

Would it be possible to either add a bit more context on how to use, or maybe have it handle some of the setup background? I'm imagining something as simple as `dest_prs.duplicate_slide(origin_prs.slides[desired_slide_index])` or something along those lines (have it be a `Presentation`-level method).

Reply:
What's in the rest of the trace? 

And the reason it's not in the presentation-level is since that would be the wrong place to put it, since it's still technically dealing with slides.

`self.presentation.slides.duplicate_slide(prs.slides[0])` was how I was running it, and it was running fine. 

I think there's something in the file that's wreaking havoc on the function and I don't know what it is.

Reply:
Updated and fixed in the next PR for when there is no "blank" layout included in the presentation